frontend/src/components/panels/Properties.vue

```diff
@@ -1,15 +1,64 @@
 <template>
-	<LayoutCol class="properties-panel"></LayoutCol>
+	<LayoutCol class="properties">
+		<LayoutRow class="options-bar">
+			<WidgetLayout :layout="propertiesOptionsLayout"></WidgetLayout>
+		</LayoutRow>
+		<LayoutRow class="sections" :scrollableY="true">
+			<WidgetLayout :layout="propertiesSectionsLayout"></WidgetLayout>
+		</LayoutRow>
+	</LayoutCol>
 </template>
 
-<style lang="scss"></style>
+<style lang="scss">
+.properties {
+	height: 100%;
+
+	.widget-layout {
+		flex: 1 1 100%;
+		margin: 0 4px;
```

Comment:
![image](https://user-images.githubusercontent.com/21245806/153620717-324c612d-3bd1-45b2-8e0c-144a867171fb.png)
When zoomed in, the margin seems to be off, is this intended?

Reply:
@Keavon ?

Reply:
Noticed that. Need to find a more robust CSS solution for the labels on the left and widgets on the right. After alpha.